src/Features/Core/Portable/ReplacePropertyWithMethods/ReplacePropertyWithMethodsCodeRefactoringProvider.cs

```diff
@@ -74,13 +75,21 @@ internal class ReplacePropertyWithMethodsCodeRefactoringProvider : CodeRefactori
                 propertyName));
         }
 
+        private static ImmutableArray<NamingRule> GetNamingRules(Document document, CancellationToken cancellationToken)
+        {
+            return document.GetNamingRulesAsync(cancellationToken).GetAwaiter().GetResult()
+                .AddRange(DefaultNamingRules.FieledAndPropertyRules);
+        }
+
         private async Task<Solution> ReplacePropertyWithMethodsAsync(
            Document document,
            IPropertySymbol propertySymbol,
            CancellationToken cancellationToken)
         {
+            var namingRules = GetNamingRules(document, cancellationToken);
             var desiredMethodSuffix = NameGenerator.GenerateUniqueName(propertySymbol.Name,
-                n => !HasAnyMatchingGetOrSetMethods(propertySymbol, n));
+                n => !HasAnyMatchingGetOrSetMethods(propertySymbol, n),
+                generateNewName: null);
 
             var desiredGetMethodName = GetPrefix + desiredMethodSuffix;
             var desiredSetMethodName = SetPrefix + desiredMethodSuffix;
```

Comment:
I'm not sure if we should apply naming rules for these methods

Reply:
Let's not do anything special here for now.